### ISSUE TITLE
stopping a specific extension from the auto update

### ISSUE THREAD

**Issue by MohammedFaragallah** [Microsoft/vscode#57591]
Some of my vs code extension update every day, and I have a limited amount of internet bandwidth, is it possible to add stopping a specific extension from the auto update?

Thanks in advance.

**Comment by sandy081** [comment]
There is no way to stop auto update per extension... but you can stop auto updating in general. Do you wish to try that or do you need auto updating per extension?

**Comment by MohammedFaragallah** [comment]
Thanks, @sandy081  for your answer but I don't really understand what you mean are you saying I can stop auto-updating in general and allow some of my extensions to auto update?

**Comment by sandy081** [comment]
@MohammedFaragallah I mean you can stop auto updating all extensions using following action

![image](https://user-images.githubusercontent.com/10746682/45355029-918b3300-b5bf-11e8-978f-9f21d0b1f9f0.png)



This will not auto update any of your extensions. But you have to manually update each when there is an update.

**Comment by MohammedFaragallah** [comment]
@sandy081 yes I understand, and that's why I'm asking for the feature to disable some extensions auto-update, the problem for me was solved but I still think this could be nice to prevent some extensions from auto update while all others can.
I have more than 70 extensions it will be sad if I had to update every single one of them manually just to stop one or two from auto-update.
